setup.py

```diff
@@ -166,61 +184,27 @@ def run(self):
         _ctx.cxx_coverage = (
             self.cxx_coverage is not False
             or bool(os.environ.get("OTIO_CXX_COVERAGE_BUILD"))
-        )
-        try:
-            out = subprocess.check_output(['cmake', '--version'])
-        except OSError:
-            raise RuntimeError(
-                "CMake must be installed to build the following extensions: "
-                + ", ".join(e.name for e in self.extensions)
-            )
-
-        if platform.system() == "Windows":
-            cmake_version = LooseVersion(
-                re.search(r'version\s*([\d.]+)', out.decode()).group(1)
-            )
-            if cmake_version < '3.1.0':
-                raise RuntimeError("CMake >= 3.1.0 is required on Windows")
+        ) and platform.system() == "Linux"
 
         self.build()
 
     def build(self):
-        _ctx.ext_dir = os.path.join(os.path.abspath(self.build_lib), "opentimelineio")
+        print("OTIO_build_ext::build is running cmake in the build location")
```

Comment:
Is this debugging information?  Or do we want to keep this message

Reply:
That was leftover from learning how pip interacts with the various classes and functions in setup.py. Removing.